components/monitoring/logging/staging/configure-splunk-log-forwarder.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+  name: instance
```

Comment:
Perhaps choose a more meaningful name

Reply:
Quoting from the documentation:
`The name of the ClusterLogForwarder CR must be instance.`

Reply:
My bad. Fixed.